deep-learning/_posts/2017-08-02-dilated_residual_network.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: review
+title:  "Dilated Residual Networks"
+tags:   deep-learning CNN
+author: Clément Zotti
+pdf:   "https://arxiv.org/pdf/1705.09914.pdf"
+
+cite:
+
+  authors: "F.Yu, V.Koltun, T.Funkhouser"
+  title:   "Dilated Residual Networks"
+  venue:   "CVPR 2017"
+---
+
+In this paper the authors show that using dilated convolution help to produce better results for image classification,
+object localisation and image segmentaion. Also the dilated convolution is kind of plug-and-play in place of regular
+convolution with minor changes in the network.
+
+The dataset used for the image classification and object localisation is [ImageNet](http://www.image-net.org/) 2012 and
+for the image segmentation it's [CityScapes](https://www.cityscapes-dataset.com/).
+
+The main idea of this papers is help the network to preserve the spatial resolution of its input.
+
+## Models
+
+The model is a regular [ResNet]({{ site.baseurl }}{% post_url /deep-learning/2017-03-16-resnet %})
+to produce baseline results for the different tasks.
+
+The first model called (**DRN-A**) removes the latest maxpooling and use only dilated convolution with different
```

Comment:
No need for parentheses

Reply:
latest maxpooling -> last maxpooling layer
use -> uses
convolution -> convolutions